employees/models.py

```diff
@@ -201,21 +228,29 @@ class Month(models.Model):
     bool_choices = (
         (True, _('Yes')),
         (False, _('No')),
-        )
+    )
     default_year = date.today().year
     default_month = date.today().month
     months_dict = {1: 'January', 2: 'February', 3: 'March', 4: 'April', 5: 'May', 6: 'June',
                    7: 'July', 8: 'August', 9: 'September', 10: 'October', 11: 'November', 12: 'December'}
-    year = models.PositiveIntegerField(_('year'),default=default_year, validators=[MaxValueValidator(9999)])
+    year = models.PositiveIntegerField(_('year'), default=default_year, validators=[MaxValueValidator(9999)])
     employee = models.ForeignKey(Employee, null=True, blank=True, on_delete=models.CASCADE)
-    month = models.IntegerField(_(u'Month'),choices=month_choices, default=default_month)
-    salary_is_paid = models.BooleanField(_(u'Paid?'),default=False, choices=bool_choices)
-    hours_worked_in_this_month = models.DecimalField(_('hours worked in this month'),decimal_places=1, max_digits=4, default=0,
-                                                    validators=[MaxValueValidator(720), MinValueValidator(0)])
-    month_is_approved = models.BooleanField(_(u'Approved?'),default=False, choices=bool_choices)
-    rate_per_hour_this_month = models.DecimalField(_('rate per hour this month'),decimal_places=2, max_digits=7, default=0,
+    month = models.IntegerField(_(u'Month'), choices=month_choices, default=default_month)
+    salary_is_paid = models.BooleanField(_(u'Paid?'), default=False, choices=bool_choices)
+    hours_worked_in_this_month = models.DecimalField(_('hours worked in this month'), decimal_places=1, max_digits=4,
+                                                     default=0,
+                                                     validators=[MaxValueValidator(720), MinValueValidator(0)])
```

Comment:
This is unreadable fix it like this:
```
hours_worked_in_this_month = models.DecimalField(
    _('hours worked in this month'), 
    decimal_places=1, 
    max_digits=4,
    default=0,
    validators=[MaxValueValidator(720), MinValueValidator(0)]
)
```
And stick to similar writhing in others parts of the code ;)